### PR TITLE
Remove trailing spaces

### PR DESCRIPTION
**Pull request by serhiy-storchaka**

### CODE REVIEW

File: Modules/_blake2/blake2s_impl.c [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #endif
 
 #include "Python.h"
```
Comment (Fidget-Spinner) [inline Modules/_blake2/blake2s_impl.c:20]: Are the changes in this file accidental?

**Review by Fidget-Spinner — APPROVED** [review]

File: Modules/_blake2/blake2s_impl.c (hunk 1 above)
Comment (serhiy-storchaka) [reply Modules/_blake2/blake2s_impl.c:20]: This is a generated file. After regenerating all I got this.

**Comment by JelleZijlstra** [comment]
Thanks for fixing this!

Is there a place we could check for trailing spaces in CI? I'd rather not have to periodically check for this and make another similar PR.

**Comment by serhiy-storchaka** [comment]
AFAIK there were scripts and Makefile targets, but they were disabled.

**Comment by erlend-aasland** [comment]
> Thanks for fixing this!
> 
> Is there a place we could check for trailing spaces in CI? I'd rather not have to periodically check for this and make another similar PR.

+1 from me. Let's open an issue over at [python/core-workflow](https://github.com/python/core-workflow).

**Pull request merged**